<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSequenceBrowserModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerSequenceBrowserModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Active browser node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ActiveBrowser">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLSequenceBrowserNode</string>
        </stringlist>
       </property>
       <property name="baseName">
        <string>SequenceBrowser</string>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InputSection">
     <property name="text">
      <string>Browsing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="qMRMLSequenceBrowserSeekWidget" name="sequenceBrowserSeekWidget"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="qMRMLSequenceBrowserPlayWidget" name="sequenceBrowserPlayWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="SynchronizedBrowsingSection">
     <property name="text">
      <string>Synchronized nodes</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_SequenceInput">
        <property name="text">
         <string>Master node</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_Sequence">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget_SynchronizedSequenceNodes">
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Playback</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Recording</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="PlottingSection">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>37</height>
      </size>
     </property>
     <property name="text">
      <string>Plotting</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_iCharting">
        <property name="text">
         <string>Enable interactive charting</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ctkVTKChartView" name="ChartView_iCharting"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="AdvancedSection">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_PlaybackItemSkippingEnabled">
        <property name="text">
         <string>Enable item skip during playback:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_PlaybackItemSkippingEnabled">
        <property name="toolTip">
         <string>If checked, items may be skipped during playback to reach the requested frame rate. If not checked then each item is displayed during playback but the playback speed may be lower than requested.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkVTKChartView</class>
   <extends>QWidget</extends>
   <header>ctkVTKChartView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSequenceBrowserPlayWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSequenceBrowserPlayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSequenceBrowserSeekWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSequenceBrowserSeekWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerSequenceBrowserModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerSequenceBrowserModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_Sequence</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSequenceBrowserModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ActiveBrowser</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
